src/course-unit/course-xblock/render-error-alert/index.jsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
+import { Info as InfoIcon } from '@edx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import AlertMessage from '../../../generic/alert-message';
+import messages from './messages';
+
+const RenderErrorAlert = ({
+  variant, icon, title, description, errorMessage, ...props
+}) => {
+  const intl = useIntl();
+
+  return (
+    <AlertMessage
+      icon={icon}
+      className="m-0"
+      variant={variant}
+      title={title || intl.formatMessage(messages.alertRenderErrorTitle)}
```

Comment:
```suggestion
      const alertTitle = title || intl.formatMessage(messages.alertRenderErrorTitle);
      ...
      title={alertTitle}
```

Reply:
I'm not sure, that is the best solution for this case because we have the same for the description prop.
I think, it looks better inline in the component instead of variable.
What do you think about it?

Reply:
I think it is not necessary here to create a new variable